mix.exs

```diff
@@ -154,7 +154,21 @@ defmodule EMQXUmbrella.MixProject do
   # need to remove those when listing `/apps/`...
   defp enterprise_umbrella_apps() do
     MapSet.new([
-      :emqx_bridge_kafka
+      :emqx_bridge_kafka,
+      :emqx_bridge_cassandra,
+      :emqx_bridge_clickhouse,
```

Comment:
The following apps (clickhouse, dynamo, ...) have been created in https://github.com/emqx/emqx/pull/10402, we forgot to update the packaging logic accordingly.